lib/commands/context.js

```diff
@@ -192,8 +194,11 @@ async function setupExistingChromedriver (chromedriver) {
 }
 
 async function setupNewChromedriver (opts, curDeviceId, adbPort) {
+  let getPort = BlueBird.promisify(PortFinder.getPort, {context: PortFinder});
+  let chromePort = await getPort();
+  logger.error(`Using ${chromePort} for new ChromeDriver session`);
   let chromeArgs = {
-    port: opts.chromeDriverPort,
+    port: chromePort,
```

Comment:
`opts.chromeDriverPort` was not being set by anything. I grepped all the code in `appium-android-driver` for that string.